src/Options/Options.hpp

```diff
@@ -164,7 +164,7 @@ struct create_from_yaml {
 /// \snippet Test_Options.cpp alternatives
 template <typename... AlternativeLists>
 struct Alternatives {
-  static_assert(sizeof...(AlternativeLists) >= 2,
+  static_assert(sizeof...(AlternativeLists) >= 1,
```

Comment:
Update the error message. Also, @wthrowe are you on board with this change? The context is that a `std::variant` is well-defined for a single type. We could also omit the `std::variant` in the option type until we implement the second type.

Reply:
Yes, he discussed it with me.  I don't think there are any technical problems with the change.  I suspect the requirement was there because I just wasn't thinking of TMP-generated option lists, or possibly because it will look weird in the help output.  The latter is probably still true, but we can fix that later if we think it's worth the effort.  (There are worse issues with help text.)